From 60aacdb5f635c747ed66c67939fe74675e401611 Mon Sep 17 00:00:00 2001
From: Yong Yao <yong.yao@intel.com>
Date: Tue, 2 Apr 2013 00:21:25 +0800
Subject: [PORT FROM MAIN]wifi: pass the build for MR2 PDK

BZ: 117349

Pass the build for MR2 PDK

Change-Id: I5fd212de8cfdebdfdc1b50543e0e15f0dbaf37f7
Orig-Change-Id: I216bb4aa02d1ff535bf784881dc5f14fbea31a4a
Category: device enablement
Domain: CWS.WIFI-Intel
Origin: internal
Upstream-Candidate: no,proprietary
Signed-off-by: Yong Yao <yong.yao@intel.com>
Signed-off-by: Jeremie GARCIA <jeremie.garcia@intel.com>
---
 Android.mk |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/Android.mk b/Android.mk
index b26ac8d..ed0238a 100644
--- a/Android.mk
+++ b/Android.mk
@@ -37,6 +37,7 @@ LOCAL_C_INCLUDES := $(KERNEL_HEADERS) \
                     external/openssl/include \
                     external/stlport/stlport \
                     bionic \
+                    system/core/logwrapper/include/ \
                     bionic/libc/private \
                     $(call include-path-for, libhardware_legacy)/hardware_legacy \
 
@@ -46,7 +47,7 @@ LOCAL_CFLAGS += -DCONFIG_LIBNL20 -Werror=format
 
 
 LOCAL_SHARED_LIBRARIES := libstlport libsysutils liblog libcutils libnetutils \
-                          libcrypto libhardware_legacy libmdnssd libdl
+                          libcrypto libhardware_legacy libmdnssd libdl liblogwrap
 
 LOCAL_STATIC_LIBRARIES := libnl_2
 
@@ -89,6 +90,7 @@ LOCAL_C_INCLUDES := $(KERNEL_HEADERS) \
                     external/mdnsresponder/mDNSShared \
                     external/openssl/include \
                     external/stlport/stlport \
+                    system/core/logwrapper/include/ \
                     bionic \
                     bionic/libc/private \
                     $(call include-path-for, libhardware_legacy)/hardware_legacy \
-- 
1.7.4.1

